opty/direct_collocation.py

```diff
@@ -110,10 +115,14 @@ def __init__(self, obj, obj_grad, *args, **kwargs):
 
         """
 
-        self.bounds = kwargs.pop('bounds', None)
-
-        self.collocator = ConstraintCollocator(*args, **kwargs)
+        self.collocator = ConstraintCollocator(
+            equations_of_motion, state_symbols, num_collocation_nodes,
+            node_time_interval, known_parameter_map, known_trajectory_map,
+            instance_constraints, time_symbol, tmp_dir, integration_method,
+            parallel
+            )
```

Comment:
Why would this one have rendered the full list correctly if it was `*args, **kwargs`?

Reply:
Sorry, I thought that was the class definition.